chapters/002-smol-internet/README.md

```diff
@@ -190,251 +149,205 @@ Let's go back to our `docker-compose.yml` and give `boudi` an additional network
+
+> **What's with those ARP requests?**
 
-> *In terminal window 3 (tara):*
+YOu may or may not see in your own session some odd looking packets identified as `ARP` in the tcpdump:
 
 ```bash
-root@tara:/# tcpdump -n
-tcpdump: verbose output suppressed, use -v[v]... for full protocol decode
-listening on eth0, link-type EN10MB (Ethernet), snapshot length 262144 bytes
-19:57:13.285340 IP 10.1.2.3 > 10.1.2.2: ICMP echo request, id 4, seq 1, length 64
-19:57:13.285392 IP 10.1.2.2 > 10.1.2.3: ICMP echo reply, id 4, seq 1, length 64
-19:57:14.321680 IP 10.1.2.3 > 10.1.2.2: ICMP echo request, id 4, seq 2, length 64
-19:57:14.321698 IP 10.1.2.2 > 10.1.2.3: ICMP echo reply, id 4, seq 2, length 64
-19:57:15.342833 IP 10.1.2.3 > 10.1.2.2: ICMP echo request, id 4, seq 3, length 64
-19:57:15.342849 IP 10.1.2.2 > 10.1.2.3: ICMP echo reply, id 4, seq 3, length 64
-19:57:16.367010 IP 10.1.2.3 > 10.1.2.2: ICMP echo request, id 4, seq 4, length 64
-19:57:16.367026 IP 10.1.2.2 > 10.1.2.3: ICMP echo reply, id 4, seq 4, length 64
-^C
-8 packets captured
-8 packets received by filter
-0 packets dropped by kernel
+18:53:54.166970 ARP, Request who-has 10.1.2.3 tell 10.1.2.2, length 28
+18:53:54.167080 ARP, Request who-has 10.1.2.2 tell 10.1.2.3, length 28
+18:53:54.167123 ARP, Reply 10.1.2.2 is-at 02:42:0a:01:02:02, length 28
+18:53:54.167168 ARP, Reply 10.1.2.3 is-at 02:42:0a:01:02:03, length 28
 ```
 
-We can see that the ping from `boudi` made it to `tara`! Huzzah!
-
-### Can `tara` ping `boudi`?
-
-Now... let's see if `tara` is able to ping `boudi`. We're going to do the same 3 window setup, but this time `tara` will be running both `ping` and `tcpdump`.
-
-The interesting thing with `tara` pinging `boudi` is that `boudi` has 2 network IP addresses we can ping. Which one should we use? Well, 10.1.1.3 is the IP that's on the `squasheeba` network. `tara`, unfortunately, doesn't know anything about how to reach this network, which means, if we try to ping it, `boudi` won't receive the ping and `tara` is just screaming into the void. If instead, we use `boudi`'s IP on `doggonet`, 10.1.2.3, we should have a successful result.
+We go over this in more detail in [ip-and-mac-addresses.md in the appendix](../appendix/ip-and-mac-addresses.md), but let's look at a high level at what's going on here. IP addresses, like `10.1.2.3`, are used by machines for identifying where packets should be routed across an internet. So what we've been working with so far is designed to help machines communicate when there are multiple networks. HOWEVER. Within a network, machines are not identified by an IP address, but instead by a MAC address. In order for packets to be forwarded from one machine on a network to another machine on the same network, each machine needs to discover the MAC address that corresponds to the IP address identified in the packets. ARP, or Address Resolution Protocol, is the process by which this is done.
```

Comment:
```suggestion
We go over this in more detail in [ip-and-mac-addresses.md in the appendix](../appendix/ip-and-mac-addresses.md), but let's look at a high level at what's going on here. IP addresses, like `10.1.2.3`, are used by machines for identifying where packets should be routed across an internet. So what we've been working with so far is designed to help machines communicate when there are multiple networks. HOWEVER. Within a network, machines are not identified by an IP address, but instead by a MAC address. In order for packets to be sent from one machine on a network to another machine on the same network, each machine needs to discover the MAC address that corresponds to the IP address identified in the packets. ARP, or Address Resolution Protocol, is the process by which this is done.
```
I get a heebie-jeebie when I see the term `forwarded` here, since it's routers that forward packets (okay, switches sort of do this too, but we don't really talk about that as "forwarding"). In the layer-2 case, we are really just putting the packet on the wire and having the other machine pick it up directly.

Reply:
Heh. I remember having a similar thought when I wrote this. I made a mental note to come back and think on what word to use here, but clearly.... I forgot 😳